screens/pages/components/albums/src/index.js

```diff
@@ -169,19 +174,20 @@ class GalleryMediaPicker extends Component {
                         picked_up: false,
                         tags: {},
                         type: 'gallery'
-                    };
-                } else {
-                    return null;
+                    });
                 }
             }
         });
+        console.log('=====');
 
         // console.log('geotagged[0]', geotagged[0]);
-        // console.log({ geotagged });
+        // console.log(JSON.stringify(geotagged, null, '\t'));
 
         let albums = [{ albumName: 'Geotagged', images: geotagged }];
-
+        // console.log(albums);
```

Comment:
Remove logs

Reply:
these files are part of old forked gallery code which is not used anymore... there are kept here just for reference.. we will delete the whole folder once the current gallery implementation is tested.